lib/media/region_observer.js

```diff
@@ -96,6 +96,18 @@ shaka.media.RegionObserver = class extends shaka.util.FakeEventTarget {
       {
         weWere: BEFORE_THE_REGION,
         weAre: AFTER_THE_REGION,
+        wasSeeking: false,
+        invoke: (region, seeking) => {
+          // This is the case of regions whose duration is smaller than the
+          // resolution of our polling.
+          this.onEvent_('enter', region, seeking);
+          this.onEvent_('exit', region, seeking);
+        },
```

Comment:
Maybe instead of new rule, change invoke implementation?
```suggestion
        invoke: (region, seeking) => {
          if (seeking) {
          this.onEvent_('skip', region, seeking);
          } else {
            // This is the case of regions whose duration is smaller than the
            // resolution of our polling.
            this.onEvent_('enter', region, seeking);
            this.onEvent_('exit', region, seeking);
          }
        },
```

Reply:
Done!